### PR TITLE
panel_18

### PR DESCRIPTION
**Pull request by Jiaqi-beep**
- NCL Script: https://www.ncl.ucar.edu/Applications/Scripts/panel_18.ncl
- NCL Plot: https://www.ncl.ucar.edu/Applications/Images/panel_18_lg.png
- Issue: #336 
panel 18

### CODE REVIEW

**Comment by Jiaqi-beep** [comment]
@anissa111 This is the one with need for accessing new datafile. Thank you!

**Comment by anissa111** [comment]
The datafile should be available now!

File: Plots/Panels/NCL_panel_18.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    # Use geocat.viz.util convenience function to set titles
+    gvutil.set_titles_and_labels(ax,
+                                 lefttitle='TS',
+                                 righttitle='°C',
+                                 lefttitlefontsize=10,
+                                 righttitlefontsize=10)
+    # Add center title
+    ax.set_title(title, loc='center', y=1.04, fontsize=10)
+
+# Import NCL colormaps
```
Comment (erinlincoln) [inline Plots/Panels/NCL_panel_18.py:100]:
```suggestion
# Import colormaps
```

**Review by erinlincoln — COMMENTED** [review]
One small edit below since all the colormaps aren't NCL. One other small thing I noticed was that the colorbar on your end has tick marks, but the original plot does not.

**Review by anissa111 — APPROVED** [review]
This looks good to me, thank you!

**Comment by Jiaqi-beep** [comment]
> One small edit below since all the colormaps aren't NCL. One other small thing I noticed was that the colorbar on your end has tick marks, but the original plot does not.

Thanks! I've removed them.

**Review by michaelavs — APPROVED** [review]
Looks good to me!

**Pull request merged**